solutions/cf-1279D.mdx

```diff
@@ -77,10 +77,11 @@ int main() {
 	for (const vector<int>& c : children) {
 		for (const int& i : c) {
 			int num = 1 * 1 * wanter_nums[i];
```

Comment:
?

Reply:
think it's part of some math formula that happens to have 1's in it
makes things more explicit